Content.Client/Chemistry/Visualizers/FoamVisualizerComponent.cs

```diff
@@ -0,0 +1,18 @@
+using Robust.Client.Animations;
+namespace Content.Client.Chemistry.Visualizers;
+
+[RegisterComponent]
+[Access(typeof(FoamVisualizerSystem))]
+public sealed class FoamVisualizerComponent : Component
+{
+    public const string AnimationKey = "foamdissolve_animation";
+
+    [DataField("animationTime")]
+    public float Delay = 0.6f;
+
+    [DataField("animationState")]
+    public string State = "foam-dissolve";
```

Comment:
make this a mapped layer or a SpriteSpecifier

Reply:
Animations do not function with sprite specifiers as far as I can tell.

Reply:
Wait, NVM the mapping is just for the animation.